elliott/elliottlib/bzutil.py

```
@@ -977,6 +980,27 @@ def create_issue(
+        """Filter version names to only those that exist in the Jira project.
+
+        Args:
+            versions: Configured version names to validate.
+
+        Returns:
+            List of version names that exist in the project. Returns the
+            original list unchanged if the project versions cannot be fetched.
+        """
+        try:
+            project_versions = {v.name for v in self._client.project_versions(self.project)}
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)